File: generatePin.go
package main

import (
	"fmt"
	"strings"
	"net/http"
	"io/ioutil"
)

func main() {

	url := "https://api.kaiza.la/v1/generatePin"

	payload := strings.NewReader("{\"mobileNumber\":\"+919910870005\", applicationId:\"37ba1e41-d55c-43bf-a2fa-d72c97aa080a\"}")

	req, _ := http.NewRequest("POST", url, payload)

	req.Header.Add("Content-Type", "application/json")
	req.Header.Add("cache-control", "no-cache")

	res, _ := http.DefaultClient.Do(req)

	defer res.Body.Close()
	body, _ := ioutil.ReadAll(res.Body)

	fmt.Println(res)
	fmt.Println(string(body))

}

File: Authentication/generatePin.py
import requests

url = "https://api.kaiza.la/v1/generatePin"

payload = "{'mobileNumber':"",'applicationId':""}"
headers = {
    'Content-Type': "application/json",
    'cache-control': "no-cache"
    }

response = requests.request("POST", url, data=payload, headers=headers)

print(response.text)
